js/task-controller.js

```diff
@@ -0,0 +1,105 @@
+export default class TaskController {
+
+    _tasks = [];
+
+    currentTasks_container = document.querySelector("#currentTasks");
+    completedTasks_container = document.querySelector("#completedTasks");
```

Comment:
use constructor for this

Reply:
created a constructor
```suggestion
    constructor(current = "#currentTasks", completed = "#completedTasks", editForm = "#edit_form") {
        this.currentTasks_container = document.querySelector(current);
        this.completedTasks_container = document.querySelector(completed);
        this.editModalForm = document.querySelector(editForm);
    }
```